typings/internal-electron.d.ts

```diff
@@ -97,6 +98,11 @@ declare namespace Electron {
     _setOwnerWindow(w: BaseWindow | null): void;
   }
 
+  interface NavigationEntry {
```

Comment:
We don't need this defined type, it is auto generated as `Electron.NavigationEntryAtIndex`